kingpin/actors/aws/ecs.py

```diff
@@ -406,7 +420,369 @@ def _execute(self):
         if self.option('wait'):
             yield self._wait_for_tasks(tasks)
         else:
-            self.log.info('Not waiting for tasks to stop')
+            self.log.info('Not waiting for tasks to complete')
+
+
+# http://boto3.readthedocs.io/en/latest/reference/services/ecs.html
+SERVICE_DEFINITION_SCHEMA = {
+    'type': 'object',
+    'additionalProperties': False,
```

Comment:
I would re-think `additionalProperties: False` -- with this in place, every time Amazon adds some new feature to ECS (Which they're doing regularly), we are forced to update our code. I would switch this to `True` here and in the TASK definition scema.